src/NServiceBus.AcceptanceTests/UnitOfWork/TransactionScope/When_enabled.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace NServiceBus.AcceptanceTests.UnitOfWork
+{
+    using System.Threading.Tasks;
+    using System.Transactions;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_enabled
```

Comment:
I see that the test naming would make sense when considering the location of the file but it doesn't work when considering the namespace (which is the information you see on the type) nor when searching for tests about transaction scope. I'd suggest to rename to "When_transaction_scope_enabled".